r_scripts/analysis/oregon_part_a.Rmd

```diff
@@ -0,0 +1,1083 @@
+  n = n+1
+}
+
+# check the number of people in each disease
+num_person <- data.frame(rbind(var_list, N))
+```
+
+Then load the Emergency data set.
+
+```{r load ER data set and zip smoke, echo=FALSE, message=FALSE, warning=FALSE}
+## Analysis
+path <- paste0("../data_new/casecrossover/ER/")
+
+resp_casecross <- read_csv(paste(path, 
+                                 "or_respiratory_may_to_sep_time_strat_casecross_er.csv", 
+                                 sep = "/"))
+
+# asthma
+asthma_casecross <- read_csv(paste(path, 
+                                   "or_asthma_may_to_sep_time_strat_casecross_er.csv", 
+                                   sep = "/")) 
+
+# copd 
+copd_casecross <- read_csv(paste(path, 
+                                 "or_copd_may_to_sep_time_strat_casecross_er.csv", 
+                                 sep = "/"))
+
+# pneum or bronchitis
+pneum_casecross <- read_csv(paste(path, 
+                                  "or_pneumonia_may_to_sep_time_strat_casecross_er.csv",
+                                  sep="/"))
+ # acute bronchitis
+acute_bronch_casecross <- read_csv(paste(path, 
+                                         "or_acute_bronch_may_to_sep_time_strat_casecross_er.csv",
+                                         sep = "/"))
+# cvd
+cvd_casecross <- read_csv(paste(path, 
+                                "or_cvd_may_to_sep_time_strat_casecross_er.csv",
+                                sep="/"))
+# arrhythmia
+arrhythmia_casecross <- read_csv(paste(path, 
+                                       "or_arrhythmia_may_to_sep_time_strat_casecross_er.csv",
+                                       sep="/"))
+# cerebral vascular
+cereb_vas_casecross <- read_csv(paste(path, 
+                                      "or_cerbrovas_dis_may_to_sep_time_strat_casecross_er.csv",
+                                      sep="/"))
+# heart failure
+hf_casecross <- read_csv(paste(path, 
+                               "or_heart_failure_may_to_sep_time_strat_casecross_er.csv", 
+                               sep="/"))
+# ischemic heart disease
+ihd_casecross <- read_csv(paste(path, 
+                                "or_isch_heart_dis_may_to_sep_time_strat_casecross_er.csv",
+                                sep="/"))
+# myo infarc
+mi_casecross <- read_csv(paste(path, 
+                               "or_myocardial_infarc_may_to_sep_time_strat_casecross_er.csv", sep="/"))
+
+
+
+```
+
+
+
+### Descriptive Table
+
+This table shows number of cardiopulmonary visits for each outcome observed from May 1st to September 30st in 2013. Sex and Age strata for each outcome are also shown in this table. Female and Male show similar proportion in each outcome. As for age groups, adult group shows most proportion in asthma outcome than other age groups, and also it is the more likely disease adults tend to have. Older people tend to get cardiovascular disease while children tend to get respiratory diseases, which makes sense.

```

Comment:
Remember to suppress cell sizes less than 20. Specifically for kids with cardiovascular outcomes.